webcomponents/core/src/components/reveal/deckdeckgo-reveal/deckdeckgo-reveal.tsx

```diff
@@ -60,6 +58,7 @@ export class DeckdeckgoReveal {
 
   render() {
     return <Host>
+      {arrayOf(this.visibleIndex + 1).map(index => <RevealNthChild index={index} />)}
```

Comment:
Where the ✨magic ✨ happens. Generates a `RevealNthChild` component (`<style/>`) for each index up to `visibleIndex + 1`;

Reply:
Pure ✨magic ✨, really more beautiful and modern than adding and removing the script "manually" 👍

Reply:
Thx again